src/main/java/com/deegeu/trivia/model/TriviaQuestionArrayAccess.java

```diff
@@ -86,7 +88,17 @@ public TriviaQuestion getRandomQuestion() {
 
     @Override
     public List<TriviaQuestion> getSpecifiedQuestionList(long... id) {
-        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
+
+        // Convert the var args into a List
+        final List<Long> ids = Arrays.stream(id)
+                                     .boxed()
+                                     .collect(Collectors.toList());
+
+        // Filter the question list to only have question with the given ids
+        return this.questionList.stream()
```

Comment:
the steam will only accept ids that are the in questions list. If an Id is invalid or doesn't exist, it isn't added to the resultant list.